src/api/kube.ts

```diff
@@ -568,24 +568,20 @@ export class KubeHelper {
     }
 
     if (!res || !res.body || !res.body.items) {
-      throw new Error(`Get pods by selector "${selector}" returned an invalid response`)
+      throw new Error(`Get pods by selector "${selector}" returned an invalid response.`)
     }
 
+    // No pods found by the specified selector. So, it's not ready.
```

Comment:
Are you sure that this is the right place for this comment?

Reply:
What do you mean by that?

Reply:
To my mind this comment should be inside `if` block. Because in the place where this comment is now it is not true that "No pods found by the specified selector".

Reply:
Fixed